engine/schema/src/main/java/com/cloud/service/ServiceOfferingVO.java

```diff
@@ -103,11 +146,16 @@ public ServiceOfferingVO(String name, Integer cpu, Integer ramSize, Integer spee
         volatileVm = false;
         this.defaultUse = defaultUse;
         this.vmType = vmType == null ? null : vmType.toString().toLowerCase();
+        uuid = UUID.randomUUID().toString();
+        this.systemUse = systemUse;
+        this.name = name;
+        this.displayText = displayText;
     }
 
-    public ServiceOfferingVO(String name, Integer cpu, Integer ramSize, Integer speed, Integer rateMbps, Integer multicastRateMbps, boolean offerHA, boolean limitCpuUse,
-                             boolean volatileVm, String displayText, ProvisioningType provisioningType, boolean useLocalStorage, boolean recreatable, String tags, boolean systemUse, VirtualMachine.Type vmType) {
-        super(name, displayText, provisioningType, false, tags, recreatable, useLocalStorage, systemUse, true);
+    public ServiceOfferingVO(String name, Integer cpu, Integer ramSize, Integer speed, Integer rateMbps, Integer multicastRateMbps, boolean offerHA,
+                             boolean limitResourceUse, boolean volatileVm, String displayText, ProvisioningType provisioningType, boolean useLocalStorage, boolean recreatable, String tags, boolean systemUse,
+                             VirtualMachine.Type vmType, String hostTag, String deploymentPlanner) {
+        //super(name, displayText, provisioningType, false, tags, recreatable, useLocalStorage, systemUse, true);
```

Comment:
As git will keep the history, we can remove the code.

Reply:
+1